File: app/src/main/java/se/hactar/movieregister/helper/UiHelper.java
package se.hactar.movieregister.helper;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

public class UiHelper {

    public static void openExternal(final Context context, final String url) {
        Intent intent = new Intent(Intent.ACTION_VIEW);
        intent.setData(Uri.parse(url));
        context.startActivity(intent);
    }
}

File: app/src/main/java/se/hactar/movieregister/helper/UiHelper.kt
package se.hactar.movieregister.helper

import android.content.Context
import android.content.Intent
import android.net.Uri

object UiHelper {

    fun openExternal(context: Context, url: String) {
        val intent = Intent(Intent.ACTION_VIEW)
        intent.data = Uri.parse(url)
        context.startActivity(intent)
    }
}
